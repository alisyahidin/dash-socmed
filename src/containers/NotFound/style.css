.not-found {
  height: 100vh;
}