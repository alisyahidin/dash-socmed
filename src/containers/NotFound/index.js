import React, { Component } from 'react'
import './style.scss'

class NotFound extends Component {
  render() {
    return (
      <div className="not-found-page d-flex align-items-center justify-content-center">
        <h1>Not Found</h1>
      </div>
    )
  }
}

export default NotFound