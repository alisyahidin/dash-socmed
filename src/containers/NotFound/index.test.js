import React from 'react'
import { shallow } from 'enzyme'
import NotFound from './index'

it('render <NotFound /> correctly', () => {
  const component = shallow(<NotFound />)

  expect(component).toMatchSnapshot()
})