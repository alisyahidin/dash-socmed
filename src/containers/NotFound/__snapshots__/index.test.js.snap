// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render <NotFound /> correctly 1`] = `
<div
  className="not-found-page d-flex align-items-center justify-content-center"
>
  <h1>
    Not Found
  </h1>
</div>
`;
